src/main/java/com/api/csis3275_backend/web/TravelBuddyController.java

```diff
@@ -0,0 +1,63 @@
+package com.api.csis3275_backend.web;
+
+import com.api.csis3275_backend.entities.TravelBuddy;
+import com.api.csis3275_backend.repositories.TravelBuddyRepository;
+import com.api.csis3275_backend.service.TravelBuddyService;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+
+@RestController
+@AllArgsConstructor
+public class TravelBuddyController {
+     TravelBuddyRepository travelBuddyRepository;
+     TravelBuddyService travelBuddyService;
+
+
+     @GetMapping(path = "/api/admin/travelBuddies")
```

Comment:
I think 'api/admin/*' should be in the adminController.
What do you think?

Reply:
I don't know, I don't think it matters

Reply:
It matters if design but I think it should be fine for this project.